code/src/pages/cart/components/CartItem.tsx

```diff
@@ -0,0 +1,92 @@
+import { useEffect, useState } from "react";
+
+import AddCircleOutlineIcon from "@mui/icons-material/AddCircleOutline";
+import DeleteIcon from "@mui/icons-material/Delete";
+import RemoveCircleOutlineIcon from "@mui/icons-material/RemoveCircleOutline";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+import formatPrice from "@/utils/format-price/formatPrice";
+
+import "@/pages/cart/components/CartItem.scss";
+
+type CartItemProps = {
+  item: {
+    image: string;
+    name: string;
+    productPrice: number;
+    quantity: number;
+    calculatedPrice: number;
+  };

```

Comment:
Same here. Create another type above. We should avoid having more that one level nesting in types

Reply:
> Same here. Create another type above. We should avoid having more that one level nesting in types

created separate `ProductCartType`